src/ranges.js

```diff
@@ -113,9 +113,100 @@ const findSoleUncommonTimeRangesEnd = function(original, update) {
   return result[0];
 };
 
+/**
+ * Calculate the intersection of two TimeRanges
+ * @param bufferA {TimeRanges}
```

Comment:
Type comes before parameter name and is followed by description (not that its really needed in this case) so it would look something like this. (some of our current doc comments do it backwards, which I will fix later)
``` JavaScript
* @param {TimeRanges} bufferA the first buffer
* @param {TImeRanges} bufferB the second buffer